pkg/util/paramtable/service_param.go

```diff
@@ -625,6 +653,122 @@ please adjust in embedded Milvus: /tmp/milvus/rdb_data`,
 	r.TickerTimeInSeconds.Init(base.mgr)
 }
 
+// NatsmqConfig describes the configuration options for the Nats message queue
+type NatsmqConfig struct {
+	Server natsmqConfigServer
+}
+
+type natsmqConfigServer struct {
+	Port              ParamItem
+	StoreDir          ParamItem
+	MaxMemoryStore    ParamItem
+	MaxFileStore      ParamItem
+	MaxPayload        ParamItem
+	MaxPending        ParamItem
+	Monitor           natsmqConfigServerMonitor
+	InitializeTimeout ParamItem
+}
+
+type natsmqConfigServerMonitor struct {
+	Debug        ParamItem
+	LogTime      ParamItem
+	LogFile      ParamItem
+	LogSizeLimit ParamItem
+}
+
+// Init sets up a new NatsmqConfig instance using the provided BaseTable
+func (r *NatsmqConfig) Init(base *BaseTable) {
```

Comment:
Set version to 2.3.0 for all new parameters

Reply:
OOC, where do we read this "version" field?

Reply:
It just indicates this parameter bringing into Milvus with this version.

Reply:
In that case, we probably don't need to have it in every single param. A global param is good enough.

For this pr, we can have a local/global variable instead of specifying "2.2.0"/"2.3.0" per param.